plugins/generic/externalFeed/ExternalFeedBlockPlugin.inc.php

```diff
@@ -17,10 +17,12 @@
 
 class ExternalFeedBlockPlugin extends BlockPlugin {
 	/** @var string Name of parent plugin */
-	var $parentPluginName;
+	protected $parentPluginName;
+	protected $parentPlugin = null;
 
-	function __construct($parentPluginName) {
+	public function __construct($parentPluginName, $plugin) {
```

Comment:
I think the `$parentPluginName` is only used to fetch the plugin, so if we're passing `$plugin` directly instead, it should be possible to get rid of `$parentPluginName`, I think?